README.md

```diff
@@ -1 +1,49 @@
-# Portfolio-build
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Personal Portfolio Page
+
+> This is the mobile version of a personal portfolio page that shows off my accomplishments as a software developer.
+
+![screenshot](assets/screenshot.PNG)
+
+## Built With
+
+- HTML and CSS
+- Bootstrap
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- A simple web browser (preferably Chrome or Firefox)
+- An IDE
+
+### Setup
+- Download this repository and open index.html file.
+
+## Authors
+
+👤 **Author**
```

Comment:
- [ ] Kindly replace Author with your name which gives a clear idea of who the author is.